Libraries/LibWeb/IndexedDB/Internal/Algorithms.cpp

```diff
@@ -2365,7 +2365,7 @@ WebIDL::ExceptionOr<GC::Ref<IDBRequest>> create_a_request_to_retrieve_multiple_i
     Bindings::IDBCursorDirection direction;
 
     // 8. If running is a potentially valid key range with queryOrOptions is true, then:
-    if (is_a_potentially_valid_key_range(realm, query_or_options)) {
+    if (query_or_options.is_nullish() || is_a_potentially_valid_key_range(realm, query_or_options)) {
```

Comment:
I can't find anything in the spec to back this up. It all comes down to ["convert a value to a key"](https://www.w3.org/TR/IndexedDB/#convert-a-value-to-a-key) supporting a `null` value, which it doesn't since it returns "invalid type" for that.

Seems like this should be a spec issue at least? Please open an upstream issue and link it in an `// AD-HOC: ..` comment above this line.

Reply:
Oh right, that's what got me confused as well - wouldn't know what `range` refers to in that note. Still weird that they'd have this as a note instead of a separate spec step, but whatever. Let's see how the spec issue holds up :)